.travis.yml

```diff
@@ -9,11 +9,21 @@ before_install:
   - sudo apt-get install -y libgdal1h gdal-bin libgdal-dev
 install:
   - pip install --install-option="--no-cython-compile" cython
-  - "pip install -r requirements-dev.txt"
-  - "pip install pytest pytest-cov"
-  - "pip install coveralls"
-  - "pip install -e ."
+  - pip install -r requirements-dev.txt
+  - pip install -e .
 script: 
   - py.test --cov rasterio --cov-report term-missing
 after_success:
   - coveralls
+before_deploy:
+  - pip wheel --wheel-dir=/tmp/wheelhouse -r requirements-dev.txt
+  - pip wheel --wheel-dir=/tmp/wheelhouse -r requirements.txt
+  - pip wheel --wheel-dir=/tmp/wheelhouse .
+  - tar -C /tmp -czvf rasterio-travis-wheels-$TRAVIS_PYTHON_VERSION.tar.gz wheelhouse
```

Comment:
@sgillies given that the wheels are built for Linux only (compiled stuff anyway), why not name them w/ ```linux_x86_64```.  I think this would make target architecture more obvious to downstream users.